frontend/nuxt.config.ts

```diff
@@ -192,7 +192,7 @@ export default defineNuxtConfig({
   plausible: {
     enabled: !isTest,
     logIgnoredEvents: !isProductionBuild,
-    trackLocalhost: !isProdNotPlaywright,
+    ignoredHostnames: [],
```

Comment:
@obulat I was thinking about this, do you think "staging.openverse.org" should be in here? And if "localhost" is in here, I am curious; are the events still logged? If yes, it might make sense to hardcode this as `["staging.openverse.org", "localhost"] here and set it as an empty array in Playwright's config. That assumes we need to turn on tracking in Playwright to properly test that the events are firing.